Dockerfile

```diff
@@ -1,4 +1,8 @@
-FROM node:16-slim AS site-build
+ARG BUILDARCH=amd64
```

Comment:
it duplicated in line 43, do we still need it here as well?

Reply:
@IgorGov `ARG`s are scoped in Dockerfile. For each `FROM` statement they are refreshed. Therefore they need to be redeclared.

Reply:
got you